client/rpcserver/handlers.go

```diff
@@ -463,4 +480,26 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + fmt.Sprintf(feePaidStr, "[fee]") + `"`,
 	},
+	loginRoute: {
+		pwArgsShort: `"appPass"`,
+		argsShort:   ``,
+		cmdSummary:  `Attempt to login to all known DEX servers.`,
```

Comment:
`known` -> `registered`? Basically the same thing but at first glance, one may wonder what makes a DEX server known, which ofc is registration to the DEX.

Reply:
Will change to registered.